web/middlewares.go

```diff
@@ -0,0 +1,29 @@
+package web
+
+import (
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+	"github.com/sirupsen/logrus"
```

Comment:
I would keep the `log "github.com/sirupsen/logrus"` import to keep consistency across all the project

Reply:
You're right, I actually forgot that